src/main/java/uk/gov/justice/hmpps/casenotes/notes/AmendCaseNoteRequest.kt

```diff
@@ -1,9 +1,11 @@
 package uk.gov.justice.hmpps.casenotes.notes
 
 import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.constraints.NotBlank
 
 @Schema(description = "Amend a Case Note")
 data class AmendCaseNoteRequest(
   @Schema(required = true, description = "Text to be appended to the case note")
+  @field:NotBlank(message = "text cannot be blank")
```

Comment:
I couldn't see a validation test for the amendments but that's a pre existing thing with the tests we inherited.